src/components/SearchApp/SearchApp.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react'
+import classes from './Container.css'
```

Comment:
in order to create a module we actually have to write module in name like container.module.css.
and filename should be lower case